Makefile

```diff
@@ -64,11 +64,6 @@ dc-up: run-composers
 		--username AWS --password-stdin 311462405659.dkr.ecr.eu-west-1.amazonaws.com; \
 	docker-compose up
 
-# target for users outside MoJ to run the stack without 3rd party integrations
-.PHONY: dc-up-out
-dc-up-out: run-composers
-	@${MAKE} dc-up ORDNANCESURVEY=-
-
```

Comment:
Is this no longer relevant then?

Reply:
No, because we have dependencies on more services (Notify, for instance), so you can't really do anything useful with the service if you aren't a member of MoJ.

Reply:
👍